docs/en/ingest-management/elastic-agent/configuration/outputs/output-kafka.asciidoc

```diff
@@ -44,6 +44,29 @@ outputs:
       verification_mode: full
 ----
 
+== Kafka output and using Logstash to index data to Elasticsearch
+
+If you are considering using {ls} to ship the data from `kafka` to {es}, please
+be aware Elastic is not currently testing this kind of setup.
+
+The structure of the documents sent from {agent} to `kafka` must not be modified by {ls}.
+We suggest disabling `ecs_compatibility` on both the `kafka` input and the `json` codec.
+
+Refer to {ls} output for {agent} for more details.
```

Comment:
```suggestion
Refer to <<logstash-output,{ls} output for {agent}>> documentation for more details.
```
This will add a link to the standalone agent [Logstash output page](https://www.elastic.co/guide/en/fleet/current/logstash-output.html).

Reply:
Lovely! Thanks